pystiche_papers/sanakoyeu_et_al_2018/_modules.py

```diff
@@ -184,3 +191,78 @@ def residual_block(channels: int) -> ResidualBlock:
     )
 
     return ResidualBlock(residual)
+
+
+def encoder(in_channels: int = 3,) -> SequentialEncoder:
+    r"""Encoder part of the :class:`Transformer` from :cite:`SKL+2018`.
+
+    Args:
+        in_channels: Number of channels in the input. Defaults to ``3``.
+
+    """
+    modules = (
```

Comment:
This seems like my comment from https://github.com/pmeier/pystiche_papers/pull/172#discussion_r477273839 applies here as well. To make this a little simpler, we can use `more_itertools.pairwise` instead of writing our own function. To add `more_itertools` as a requirement, add it to 

https://github.com/pmeier/pystiche_papers/blob/0b8dfbd0eb67c0f40059c28a0edd82a34c0e0581/setup.cfg#L27-L30